lib/components/map/enhanced-stop-marker.js

```diff
@@ -1,14 +1,15 @@
-// FIXME: Typescript once dependencies are typed
+// TYPESCRIPT TODO: all props here are missing types
 /* eslint-disable react/prop-types */
-import { Styled as BaseMapStyled } from '@opentripplanner/base-map'
+// Modecolors is never getting passed to component, so it's always the same color
```

Comment:
Remove this comment.

Reply:
Removed in d3575aa5c7d8ea8b45d8959c9d74f64ac03b0274